src/components/widgets/timeSeries/TSBasicNumeric/TSBasicNumeric.tsx

```diff
@@ -12,13 +12,19 @@ const TSBasicNumeric: FC<TSBasicNumericProps> = (
 ) => {
   const { name, value, unit } = props;
 
+  const isFloat = (n: number | string) => {
+    return Number(n) === n && n % 1 !== 0;
+  };
+
+  const roundValue = () => (isFloat(value) ? Number(value).toFixed(2) : value);
+
```

Comment:
There must be built in functions for this. Did you check?

Reply:
Yes. There is no built in function to check the type and round it only if its a number